### PR TITLE
The admin outfit "Debug outfit" gets the breathless DNA string active now.

### PR DESCRIPTION
**Pull request by 1080pCat**
## What Does This PR Do

Title. 
## Why It's Good For The Game

Reincarnating as a vox and needing to turn breathless on constantly has frustrated me enough to make a pr about it, hopefully im not alone in this frustration. 
Yes im aware that they have a air tank, but for plasmen and vox that cant breath oxygen, they start taking damage, and is just annoying enough to have me make a pr about it.

## Testing

Tested, worked.

NPFC

### CODE REVIEW

File: code/datums/outfits/outfit_debug.dm [hunk 1]
```diff
@@ -36,6 +36,10 @@
 	. = ..()
 	if(visualsOnly)
 		return
+	H.dna.SetSEState(GLOB.breathlessblock,1)
```
Comment (Henri215) [inline code/datums/outfits/outfit_debug.dm:39]:
```suggestion
	H.dna.SetSEState(GLOB.breathlessblock, 1)
```

File: code/datums/outfits/outfit_debug.dm [hunk 2]
```diff
@@ -36,6 +36,10 @@
...
+	singlemutcheck(H, GLOB.breathlessblock, MUTCHK_FORCED)
+	H.dna.default_blocks.Add(GLOB.soberblock)
```
Comment (DGamerL) [inline code/datums/outfits/outfit_debug.dm:41]: So they're sober aswell?

Comment (1080pCat) [reply code/datums/outfits/outfit_debug.dm:41]: oh woops

File: code/datums/outfits/outfit_debug.dm [hunk 3]
```diff
@@ -36,6 +36,10 @@
 	. = ..()
 	if(visualsOnly)
 		return
+	H.dna.SetSEState(GLOB.breathlessblock, 1)
+	singlemutcheck(H, GLOB.breathlessblock, MUTCHK_FORCED)
+	H.dna.default_blocks.Add(GLOB.breathlessblock)
+	H.check_mutations = 1
 	var/obj/item/card/id/I = H.wear_id
 	if(istype(I))
 		apply_to_card(I, H, get_all_accesses(), "Debugger", "admin")
```
Comment (Contrabang) [inline code/datums/outfits/outfit_debug.dm:45]: Just for consistency with how other outfits tend to be
```suggestion
	var/obj/item/card/id/I = H.wear_id
	if(istype(I))
		apply_to_card(I, H, get_all_accesses(), "Debugger", "admin")
		
	H.dna.SetSEState(GLOB.breathlessblock, 1)
	singlemutcheck(H, GLOB.breathlessblock, MUTCHK_FORCED)
	H.dna.default_blocks.Add(GLOB.breathlessblock)
	H.check_mutations = 1
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by Contrabang — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]
> 🆑
NPFC
/:cl:

Just replace this entire block with `NPFC`

**Review by Contrabang — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**